Emby.Server.Implementations/ScheduledTasks/Tasks/PluginUpdateTask.cs

```diff
@@ -95,25 +98,18 @@ public async Task Execute(CancellationToken cancellationToken, IProgress<double>
             progress.Report(100);
         }
 
-        /// <inheritdoc />
-        public string Name => "Update Plugins";
+        public string Name => _localization.GetLocalizedString("TaskUpdatePlugins");
 
-        /// <inheritdoc />
-        public string Description => "Downloads and installs updates for plugins that are configured to update automatically.";
+        public string Description => _localization.GetLocalizedString("TaskUpdatePluginsDescription");
 
-        /// <inheritdoc />
-        public string Category => "Application";
+        public string Category => _localization.GetLocalizedString("TasksCategoryApplication");
 
-        /// <inheritdoc />
```

Comment:
These <inheritdoc/> tags should not be removed

Reply:
Thought they were useless. Restoring them.

Reply:
Thought they were useless. Restoring them.